services/slam/server.go

```diff
@@ -70,7 +71,23 @@ func (server *subtypeServer) GetPosition(ctx context.Context, req *pb.GetPositio
 func (server *subtypeServer) GetPositionNew(ctx context.Context, req *pb.GetPositionNewRequest) (
 	*pb.GetPositionNewResponse, error,
 ) {
-	return nil, errors.New("unimplemented stub")
+	ctx, span := trace.StartSpan(ctx, "slam::server::GetPosition")
```

Comment:
[nit] technically GEtPositionNew but I know it will change in the future so up to you on correcting now or letting it ride until the switch

Reply:
going to leave it as is for now unless people really want to change it to GetPositionNew